packages/client/src/clients/guide/client.ts

```diff
@@ -49,6 +49,13 @@ const DEFAULT_ORDER_RESOLUTION_DURATION = 50; // in milliseconds
 // trigger subscribed callbacks.
 const DEFAULT_COUNTER_INCREMENT_INTERVAL = 30 * 1000; // in milliseconds
 
+// Return the global window object if defined, so to safely guard against SSR.
+const checkForWindow = () => {
+  if (typeof window !== "undefined") {
+    return window;
+  }
+};
```

Comment:
@cjbell @connorlindsey Quick heads up - I pushed a [new commit](https://github.com/knocklabs/javascript/pull/711/commits/9d9de6282b8b974ca57d54e661f4ccc0b5db1520) to add this helper and use it in the guide client.

Doing a bit more reading last night, it sounds like referencing `window` object directly could be problematic in SSR as no such reference event exists. So using this helper I'm binding the return val to a new var `win`, and using that instead.